phpmyfaq/admin/assets/scss/_variables.scss

```
@@ -1,31 +1,31 @@
 //
-// phpMyFAQ 4.0 color variables
+// phpMyFAQ 4.x color variables
 //
 
 $theme: 'phpMyFAQ v4 Admin' !default;
 
 $white: #fff !default;
```

Stylelint (17.11.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)